src/parser/parsers/heroes.py

```diff
@@ -32,7 +32,7 @@ def run(self):
                 hero_stats.update(
                     self._map_attr_names(hero_value['m_mapStartingStats'], maps.get_hero_attr)
                 )
-
+                hero_stats['StatsUI'] = self._parse_stats_ui(hero_value)
```

Comment:
Firstly, you have not run the parser, which is why this change is not reflected on in the output-data.
Secondly, this appends the exact same data to every hero object. Is this needed?

Reply:
completely reworked